faf-commons-data/src/main/java/com/faforever/commons/replay/shared/LuaTable.java

```diff
@@ -0,0 +1,11 @@
+package com.faforever.commons.replay.shared;
```

Comment:
Did you mean to rename this to LuaTable? Might be a little confusing with.

Reply:
Oh, I forgot - it can also not be a table. I'll rename it back

Reply:
Reverted the rename in 7d4ef66aa3e455d10836098f9f6d0600714ba420